plot/index.html

```diff
@@ -0,0 +1,69 @@
+<html id="plot">
+
+<head>
+  <script src='../../chaise/chaise-config.js'></script>
+  <script src='../../chaise/scripts/vendor/jquery-1.11.1.min.js'></script>
+  <script src='../../chaise/scripts/vendor/bootstrap-3.3.7.min.js'></script>
+  <script src='vendor/plotly-latest.min.js'></script>
+  <script src='vendor/moment.min.js'></script>
```

Comment:
These 2 libraries are available in chaise as well, so you should include them based on the relative chaise path instead. Then remove them from the deriva-webapps/plot folder.